src/trace.c

```diff
@@ -1107,6 +1107,14 @@ trace_add_path (const void *parent,
 
 		pathname = resolved_pathname;
 
+		/* Prevent resolved_path_name from going above
+		 * PACK_PATH_MAX.
+		 */
+		if (strlen (resolved_pathname) > PACK_PATH_MAX) {
```

Comment:
Just in case: Is `>` correct here instead of `>=`? I'm wondering if the exact length of  `PACK_PATH_MAX` is ok. In addition, does it take the null string at the end into consideration?